### PR TITLE
Added isPrototypeOf

### PR DESCRIPTION
**Pull request by srinivasdamam**

### CODE REVIEW

File: documentation/README.md [hunk 1]
```diff
@@ -871,13 +872,10 @@ and returns `undefined` if the `predicate` is false.
 ```js
 const ifThen = require('1-liners/ifThen');
 
-const eq = (a, b) => a === b;
-const add = (a, b) => a + b;
-const sub = (a, b) => a - b;
```
Comment (hemanth) [inline documentation/README.md]: ?

File: documentation/README.md [hunk 2]
```diff
@@ -871,13 +872,10 @@ and returns `undefined` if the `predicate` is false.
...
-
 const words = ifThen((str) => typeof str === 'string', (str) => str.split(' '));
 
 words('Hello ES2015'); // => ['Hello', 'ES2015']
+    words(['Hello', 'ES2015']); // => undefined
```
Comment (hemanth) [inline documentation/README.md]: ?

File: documentation/README.md (hunk 1 above)
Comment (srinivasdamam) [inline documentation/README.md]: Don't why it is getting deleted while i commit. But this piece code is not no the master branch.

File: documentation/README.md (hunk 2 above)
Comment (srinivasdamam) [inline documentation/README.md]: Same as mentioned above. Getting added while i commit. Any help?

File: documentation/README.md (hunk 1 above)
Comment (hemanth) [inline documentation/README.md]: [this](https://github.com/1-liners/1-liners/pull/164/files/d14dccb2f1ee91f4621c5eaea84b2041396c8afc#r74090305) too?

**Comment by hemanth** [comment]
Would you like to rebase this?

**Comment by srinivasdamam** [comment]
Yes, updated.

**Pull request merged**